src/osd/osd_types.cc

```diff
@@ -675,6 +679,13 @@ void osd_stat_t::decode(ceph::buffer::list::const_iterator &bl)
       hb_pingtime[osd] = ifs;
     }
   }
+  if (struct_v >= 15) {
```

Comment:
Oops, the version in `DECODE_START_LEGACY_COMPAT_LEN()` is kept unchanged.

Reply:
Good catch!